### ISSUE TITLE
The background color of the collapse icon is inconsistent with the focused Actions/Properties under HC-modes

### ISSUE THREAD

**Issue by v-xuanzh** [Microsoft/AzureStorageExplorer#123]
**Storage Explorer Version**: 1.1.0 RC

**Platform**: Windows

**Architecture**: x64

**Build Number**: 20180501.4

**Commit**: 754b96f7

**Regression From**: Not a regression

#### Steps to Reproduce: ####

1. Launch Storage Explorer and expand storage accounts.
2. Change the current theme to HC-Black/Light.
3. Focus on the collapse icon in 'Actions/Properties' Panel and check its background color.

#### Expected Experience: ####

The background color of the collapse icon is consistent with the focused Actions/Properties.

#### Actual Experience: ####

The background color of the collapse icon is inconsistent with the focused Actions/Properties.
![color](https://user-images.githubusercontent.com/34729022/39517844-a3daadc4-4df0-11e8-9417-d3c3e5f75bc8.png)
